app/assistify/migrations/server/startup/migrations.js

```diff
@@ -120,15 +120,15 @@ Meteor.startup(() => {
 				};
 
 				if (msg.attachments && msg.attachments.length) {
-					if (msg.roles && msg.roles.length) {
-						// Threads created from the context menu will include the users from the parent channel.
+					if (msg.urls && msg.urls[0].url) {
```

Comment:
I am bit doubtful, whether ```msg.urls``` can be used to differentiate threads from side nav and context-menu

Reply:
It does not need to differentiate where the thread originated from. In fact, all our threads started had the first message quoted. We didn't make this difference. But it doesn't matter: If there's a quotable message, refer to it.